compose/material3/material3-window-size-class/build.gradle

```diff
@@ -14,21 +14,29 @@
  * limitations under the License.
  */
 
+import androidx.build.AndroidXComposePlugin
+import androidx.build.JetbrainsAndroidXPlugin
 import androidx.build.LibraryType
-import androidx.build.PlatformIdentifier
 
 plugins {
     id("AndroidXPlugin")
     id("com.android.library")
```

Comment:
```suggestion
```

Reply:
? the plugin is applied in other same muodules, e.g. `material3`

Reply:
nvm, mixed up with `org.jetbrains.kotlin.android`